tests/suites/test_suite_ctr_drbg.function

```diff
@@ -51,50 +144,57 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE */
-void ctr_drbg_validate_pr( data_t * add_init, data_t * entropy,
-                           data_t * add1, data_t * add2,
-                           data_t * result_str )
+void ctr_drbg_validate( int reseed_mode, char *nonce_string,
```

Comment:
Please move this function to the header section, create a fourth wrapper function for `RESEED_FIRST`, and call the wrapper functions from the `.data` file instead of calling `ctr_drbg_validate` directly.